src/components/LoanCards/NewHomePageLoanCard.vue

```diff
@@ -339,6 +339,10 @@ export default {
 				this.isLoading = false;
 			}
 		},
+		dedicationClickEvent() {
+			this.$emit('dedication-click', { loanId: this.loanId, dedicationCopy: this.dedicationsCopy });
```

Comment:
Does this get logged successfully with the router push right after?

Reply:
Works fine locally.